test/Calendar.test.js

```diff
@@ -17,23 +17,19 @@ const config = {
 describe("Calendar", () => {
 
     it("should throw if config object is not passed", () => {
-        const Calendar = createCalendar(() => {
-            return Promise.resolve(fixtures.response);
-        });
-
-        function calendar() {
-            return new Calendar();
-        }
+        const response = fixtures.getCtagResponse;
+        const request = Promise.resolve(response);
+        const Calendar = createCalendar(() => request);
 
-        expect(calendar).to.throw(Error, /Missing config object/);
+        expect(() => new Calendar()).to.throw(Error, /Missing config object/);
     });
 
     describe(".getCtag()", () => {
 
         it("should return an object with information about the calendar", () => {
-            const Calendar = createCalendar(() => {
-                return Promise.resolve(fixtures.getCtagResponse);
-            });
+            const response = fixtures.getCtagResponse;
+            const request = Promise.resolve(response);
+            const Calendar = createCalendar(() => request);
             const calendar = new Calendar(config);
 
             return calendar
```

Comment:
I think it is not necessary to catch an (re)-throw the error, because mocha should handle that. Am I wrong here?